docs/backends.rst

```diff
@@ -0,0 +1,211 @@
+===================================
+ Proctoring backend implementation
+===================================
+
+Proctoring providers (PS) who wish to integrate with Open edX should implement a `REST API`_ and a thin `Python wrapper`_, as described below.
```

Comment:
Oops. I did have them all as PP, but decided to change it to PS for "proctoring service", to avoid the scatological references everywhere. But then I didn't change the word it stood for! haha.